package.json

```diff
@@ -712,7 +723,7 @@
         "@types/mocha": "^2.2.48",
         "@types/node": "^7.10.9",
         "@types/request-promise-native": "^1.0.17",
-        "@types/vscode": "^1.25.0",
+        "@types/vscode": "^1.40.0",
```

Comment:
Do we need to do this?  We should not increase this beyond the current minimum supported version of VS Code.

Reply:
This is required to be able to copy URL to clipboard, will separate it to a separate PR.